framework/src/utils/FunctionMaterialPropertyDescriptor.C

```diff
@@ -68,7 +70,8 @@ FunctionMaterialPropertyDescriptor<is_ad>::FunctionMaterialPropertyDescriptor(
     _derivative_symbols(rhs._derivative_symbols),
     _value(nullptr),
     _parent(parent),
-    _property_name(rhs._property_name)
+    _property_name(rhs._property_name),
+    _required(false)
```

Comment:
What's the purpose of having `_required` default to false?  In this case particularly, why is it not `rhs._required`?

Reply:
Because we copy existing descriptors to build the derivatives. We should only enforce properties to exist if they are explicitly coupled (in the `material_property_names` list).